_posts/2017/2017-12-11-set-title-in-the-view-template-using-content-for.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Set your page title in the view template using content_for'
+description: 'Cut out all that instance variable nonsense.'
+layout: article
+category: ruby
+image:
+  base: '2017/set-title-in-the-view-template-using-content-for'
+  alt: 'Books'
+  credit: 'Annie Spratt'
+  source: 'https://unsplash.com/photos/-9vMBjrU-RA'
+---
+
+Rails has `content_for` as it’s primary way to store content in one place for use in other views, layouts or helpers. See `content_for` in the Rails documentation at [in the ActionView helpers](http://api.rubyonrails.org/v5.1.3/classes/ActionView/Helpers/CaptureHelper.html#method-i-content_for).
+
+## Instead of…
+
+...using an instance variable in the controller.
```

Comment:
I would replace the full-stop at the end of this line with a colon.

Reply:
Emphasis on 'I'.

(Especially as it's not a complete sentence. You're trying to say: "Instead of X, use Y." Not: "Instead of X. Use Y." But...maybe you are! :P )